models/staging/stg_history_ledgers.sql

```diff
@@ -1,3 +1,8 @@
+{{ config(
+    tags = ["enriched_history_operations"]
```

Comment:
shouldn't this be part of the `current_state` tag as well?

Reply:
Opting to remove tags from staging